api-report/odsp-driver-definitions.api.md

```diff
@@ -179,19 +180,58 @@ export interface OdspResourceTokenFetchOptions extends TokenFetchOptions {
 // @public
 export interface ShareLinkInfoType {
     createLink?: {
-        type?: ShareLinkTypes;
-        link?: string;
+        type?: ShareLinkTypes | SharingLinkKind;
+        link?: string | SharingLink;
         error?: any;
+        shareId?: string;
     };
     sharingLinkToRedeem?: string;
 }
 
-// @public
+// @public @deprecated (undocumented)
 export enum ShareLinkTypes {
     // (undocumented)
     csl = "csl"
 }
 
+// @public
+export interface SharingLink {
```

Comment:
Can this just extend the  ShareLinkKind interface as webUrl is extra and scope/type is same as in ShareLinkKind.